asciidoc-java/src/main/java/io/yupiik/asciidoc/renderer/html/AsciidoctorLikeHtmlRenderer.java

```diff
@@ -299,7 +297,6 @@ public void visitParagraph(final Paragraph element) {
     @Override
     public void visitHeader(final Header header) {
         if (header.attributes().get("notitle") == null &&
-                !Boolean.parseBoolean(configuration.getAttributes().getOrDefault("noheader", "false")) &&
```

Comment:
this one looks ackward to me and will have side effects so minimum would be to have a renderer toggle to restore this behavior IMHO